Permanent/Models/Data/LocnVO.swift

```diff
@@ -15,17 +23,37 @@ struct LocnVO: Model {
     let postalCode, adminOneName, adminOneCode: JSONAny? // TODO
     let adminTwoName, adminTwoCode: JSONAny? // TODO
     let country, countryCode, geometryType: String?
-    let latitude, longitude, boundSouth, boundWest: Double?
+    let anyLatitude, anyLongitude: JSONAny?
+    let boundSouth, boundWest: Double?
     let boundNorth, boundEast: Double?
     let geometryAsArray, geoCodeType, geoCodeResponseAsXML: String?
     let timeZoneID: Int?
     let status, type, createdDT, updatedDT: String?
-
+    
+    var latitude: Double? {
+        if let latitude = anyLatitude?.value as? Double {
+            return latitude
+        } else if let latitude = anyLatitude?.value as? String {
+            return Double(latitude)
+        }
+        return nil
+    }
+    
+    var longitude: Double? {
+        if let longitude = anyLongitude?.value as? Double {
+            return longitude
+        } else if let longitude = anyLongitude?.value as? String {
```

Comment:
is this validation still required?

Reply:
both longitude and latitude will be treated as Double type.

Reply:
seems that longitude and latitude still are still sent in Double or String type, for the moment this validation is needed.